cli/run.go

```diff
@@ -27,9 +27,30 @@ const (
 	depManagerTryCheck = "_"
 )
 
-// --------------------
-// Utility
-// --------------------
+var runCommand = cli.Command{
+	Name:    "run",
+	Aliases: []string{"r"},
+	Usage:   "Runs a specified Workflow.",
+	Action:  run,
+	Flags: []cli.Flag{
+		// cli params
+		cli.StringFlag{Name: WorkflowKey, Usage: "workflow id to run."},
+		cli.StringFlag{Name: ConfigKey + ", " + configShortKey, Usage: "Path where the workflow config file is located."},
+		cli.StringFlag{Name: InventoryKey + ", " + inventoryShortKey, Usage: "Path of the inventory file."},
+		cli.BoolFlag{Name: "filtering", Usage: "Enables secret filtering", EnvVar: "BITRISE_SECRET_FILTERING"},
```

Comment:
please write a little more descriptive summary: `Enables secret filtering`

for example:
`Hide secret values from the log`